gigareview/common/models/EMReportJob.php

```diff
@@ -87,8 +87,15 @@ public function storeManuscripts(array $manuscripts): bool
     {
         $storeStatus = 0;
         foreach ($manuscripts as $manuscript) {
-            if ($manuscript->save()) {
-                $storeStatus = 1;
+            # Check all manuscript attribute values are valid before saving
+            if ($manuscript->validate()) {
```

Comment:
Hi @pli888,

Since the manuscript object will be validated at this step, if the sftp server config in `gigareview/common/config/params-local.php` changed to
```
    'sftp' => [
        "host" => "sftp_test",
        "username" => "testuser",
        "password" => "testpass",
        "baseDirectory" => "editorialmanager",
    ],
```
and when executing the manuscript-q job, would get the validation error:
```
 % docker-compose run --rm console ./yii manuscripts-q/run`
Creating review_console_run ... done
Error: Manuscript GIGA-D-22-00060 failed validation
array(1) {
  ["editorial_status"]=>
  array(1) {
    [0]=>
    string(28) "Editorial Status is invalid."
  }
}
Error: Manuscript GIGA-D-22-abcde failed validation
array(1) {
  ["manuscript_number"]=>
  array(1) {
    [0]=>
    string(29) "Manuscript Number is invalid."
  }
}
Error: Manuscript GIGA-D-22-00099 failed validation
array(1) {
  ["editorial_status_date"]=>
  array(1) {
    [0]=>
    string(47) "The format of Editorial Status Date is invalid."
  }
}
```
So, maybe is it better to remove those invalid data in `gigareview/console/tests/_data/Report-GIGA-em-manuscripts-latest-214-20220607004243.csv` which were meant to test the validation rules in `manuscript` model class.

Reply:
@kencho51 I've corrected the validation errors in the CSV file. I've added documentation on how to use the `sftp_test` service into `README.md`.

After looking at my code again, I realise acceptance tests for manuscript records on the manuscript index page are missing from this PR.